app/scripts/modules/google/src/autoscalingPolicy/components/basicSettings/basicSettings.component.html

```diff
@@ -49,4 +49,22 @@
         Must be greater than or equal to 15 seconds.
       </div>
   </div>
+  <div class="row">
+    <div class="col-md-3 sm-label-right">
+      Mode
+      <help-field key="gce.serverGroup.autoscaling.mode"></help-field>
+    </div>
+    <div class="col-md-2 content-fields">
+      <select class="form-control input-sm"
+              name="mode"
+              ng-model="$ctrl.policy.mode">
+        <option ng-repeat="mode in $ctrl.modes"
+                value="{{mode}}"
+                ng-selected="$ctrl.policy.mode === mode">{{mode}}</option>
+      </select>
+    </div>
+    <div class="col-md-4 error-message" ng-if="basicSettingsForm.coolDownPeriodSec.$error.min">
```

Comment:
I think this is tied to the previous form field ("Cool down period seconds") and can be removed.